### PR TITLE
Test data to test data

### PR DESCRIPTION
**Pull request by SaitejaUtpala**
* TestData ** - > **TestData
* fn_data -> fn_test_data

### CODE REVIEW

**Comment by ninamiolane** [comment]
This PR does not pass the tests. Reviewers should not be requested.

**Comment by SaitejaUtpala** [comment]
> This PR does not pass the tests. Reviewers should not be requested.

@ninamiolane  Sorry, intention was not for merging. But for actual design changes. So kindly review if this matches your expectations

File: tests/tests_geomstats/test_connection.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 class TestConnection(TestCase, metaclass=Parametrizer):
     class TestDataConnection(TestData):
```
Comment (ninamiolane) [inline tests/tests_geomstats/test_connection.py:18]: This should be ConnectionTestData

**Review by ninamiolane — COMMENTED** [review]
This design works, thanks! Some class names have not been changed though, e.g. ConnectionTestData

**Review by ninamiolane — APPROVED** [review]
Great, thanks!

**Pull request merged**